### ISSUE TITLE
Version 0.2.4 released without the fixes mentioned in the release notes

### ISSUE THREAD

**Issue by 2m** [lightbend/paradox#41]
[0.2.4 tag](https://github.com/lightbend/paradox/tree/v0.2.4) (and the jars in the maven central) does not have the fixes mentioned in the release docs.

```
* 5b38595 (HEAD -> master, origin/master, origin/HEAD) 0.2.4
* dd99f80 0.2.3 notes
* ae3ac26 0.2.2 notes
*   3c7a5dd Merge pull request #34 from gsechaud/refErrorMessage
|\  
| * b78916d Fix and add tests
| * 420aabd Fix ref bug with anchor
* |   93818c2 Merge pull request #30 from gsechaud/fixTheme
|\ \  
| |/  
|/|   
| * 2f14457 Modify unexpected behaviour of property getter
| * 17ac0f9 Add directory in template caching
| * 7ab8fc3 Fix css/js theme generation bug
| | * 1900af5 (tag: v0.2.4) 0.2.4
| | * 20bc85b 0.2.3 notes
| | * d94de8d 0.2.2 notes
| |/  
|/|   
* | bca0487 0.2.4-SNAPSHOT
* | 002a20b (tag: v0.2.3) 0.2.3
* |   fbcc979 Merge pull request #32 from jrudolph/patch-1
|\ \  
| |/  
|/|   
| * a7149d9 disable pegdown HARDWRAPS option, fixes #31
|/  
*   0fdbdac Merge pull request #28 from gsechaud/propFrontMatter
```

**Comment by eed3si9n** [comment]
oops did I forget to pull before tagging?

**Comment by eed3si9n** [comment]
Published 0.2.5 - https://github.com/lightbend/paradox/releases/tag/v0.2.5
